src/adagrams.js

```diff
@@ -1,15 +1,161 @@
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
-};
+  const LETTERPOOL = {
```

Comment:
This can be declared outside of the function so it does not have to be recreated each time the function is called.